docs/v1/P4Runtime-Spec.mdk

```diff
@@ -686,10 +687,11 @@ follows:
   for each pair of (`device_id`, `role_id`) at any point of time.
 
 * The streaming channel between the controller and the server defines the
-  liveness of the controller session. The controller is considered "offline" or
-  "dead" as soon as its corresponding stream channel to the switch is
-  broken, in which case the P4Runtime server quickly sets one of the slave
-  controllers with the highest `election_id` as master.
+  liveness of the controller session. The controller is considered “offline” or
+  “dead” as soon as its corresponding stream channel to the switch is broken, in
```

Comment:
could we use straight quotes for "offline" and "dead", just like in the rest of the doc?

Reply:
@stefanheule could you take care if this when you update the PR?

Reply:
Done.